camille-bot/worker-configuration.d.ts

```diff
@@ -0,0 +1,11 @@
+import { KVNamespace } from '@cloudflare/workers-types';
+
+// Generated by Wrangler by running `wrangler types`
+
+interface Env {
+	kv: KVNamespace;
+	API_HOST: string;
+	SLACK_API_TOKEN: string;
+	SLACK_COMMUNITY_ID: string;
+	SLACK_SIGNING_SECRET: string;
```

Comment:
This is how you define/reference the environment variables in a type safe way. The strings will all be referenced from your `.dev.vars` file